### PR TITLE
Rennie branch

### PR DESCRIPTION
**Pull request by ericmrennie**
webpack, redux, beginnings of react

### CODE REVIEW

**Review by EricDalio — APPROVED** [review]
All looks good. Assuming you sorted the webpack issues.

File: build/index.bundle.js.LICENSE.txt
Comment (simk209) [inline build/index.bundle.js.LICENSE.txt:1]: put in git ignore

**Review by EivindDelFierro — APPROVED** [review]
LGTM

**Review by simk209 — APPROVED** [review]
Amazing Work!

**Comment by alanaherlands** [comment]
Nice work!

**Pull request merged**